project/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,18 @@
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login-url")
```

Comment:
why do you have a LoginServlet and LogoutServlet? can you combine both and call it from the same web servlet and just return the correct login/logout? does that cause other problems

Reply:
I think its cleaner this way but it might be personal preference.